BikeData/src/main/scala/by/itechart/internship/config/FlywayConfig.scala

```diff
@@ -0,0 +1,15 @@
+package by.itechart.internship.config
+
+import org.flywaydb.core.Flyway
+
+object FlywayConfig {
```

Comment:
Not sure why you're manually calling flyway.migrate during app startup, since you've already added sbt-flyway dependency.
You could apply migration manually at any point of time using "sbt flywayMigrate" command
In real world you should avoid changing DB schema as a side effect of any other action(like app startup)